src/fabric_cli/utils/fab_cmd_set_utils.py

```diff
@@ -9,6 +9,20 @@
 from fabric_cli.utils import fab_jmespath as utils_jmespath
 
 
+def validate_item_query(query_value: str) -> None:
+    if not (
+        query_value in fab_constant.ITEM_SET_ALLOWED_METADATA_KEYS
+        or query_value == fab_constant.ITEM_QUERY_DEFINITION
+        or query_value.startswith(f"{fab_constant.ITEM_QUERY_DEFINITION}.")
+        or query_value.startswith(f"{fab_constant.ITEM_QUERY_PROPERTIES}.")
+    ):
+        raise FabricCLIError(
+            f"Invalid query '{query_value}'. Allowed queries for items are: {', '.join(fab_constant.ITEM_SET_ALLOWED_METADATA_KEYS)}, "
+            f"'{fab_constant.ITEM_QUERY_DEFINITION}', '{fab_constant.ITEM_QUERY_DEFINITION}.*', or '{fab_constant.ITEM_QUERY_PROPERTIES}.*'",
+            fab_constant.ERROR_INVALID_INPUT,
```

Comment:
do you think it will be clearer for users if we return `ERROR_INVALID_QUERY` or `ERROR_INVALID_QUERY_PARAM`?